go.mod

```diff
@@ -0,0 +1,44 @@
+module github.com/mendersoftware/deviceauth
+
+go 1.13
+
+require (
```

Comment:
Maybe we should add `go mod vendor` so that you don't need to pull all requirements on `go build`.

Reply:
> Nice! Have you tried comparing the I/O throughput using Redis vs Mongo?
> This looks good to me, I only have a short comment regarding the `go mod` migration.

On my local machine, with a single device and docker-compose, using Redis as cache makes token verification 20% faster. The real advantage is with big databases (real world scenario), especially if you consider Redis will run in Kubernetes while mongodb is hosted. In those scenarios, I expect at least 50% faster response times and a huge difference in mongodb load (which is the biggest issue we have nowadays). It will also reduce data traffic (which we pay) between Kubernetes and Atlas.

Reply:
Regarding the mod vendor, to be honest I've done it just because I failed to govendor the redis client library. I don't know what we'll decide about `go mod vendor`, we'll keep it consistent with other packages. :)